### PR TITLE
maintainers: Add redhawk as a maintainer to Plexamp

### PR DESCRIPTION
**Pull request by Redhawk18**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Redhawk18** [comment]
I know there are whitespace changes, Its automatic by my editor.

**Comment by Redhawk18** [comment]
supersedes #313499

**Review by Aleksanaa — COMMENTED** [review]
Please split into two commits:
`maintainers: add redhawk`
`plexamp: add redhawk to maintainers`

**Review by Aleksanaa — COMMENTED** [review]
Please note: The uppercase and lowercase characters of package names are different. If the first letter is capitalized, ofborg build will not be triggered, although it may not be needed here.

File: pkgs/applications/audio/plexamp/default.nix [hunk 1]
```diff
@@ -31,7 +32,7 @@ in appimageTools.wrapType2 {
     homepage = "https://plexamp.com/";
     changelog = "https://forums.plex.tv/t/plexamp-release-notes/221280/53";
     license = licenses.unfree;
-    maintainers = with maintainers; [ killercup synthetica ];
+    maintainers = with maintainers; [ killercup redhawk synthetica ];
```
[the remark this answers is not in the document]
Comment (Aleksanaa) [reply pkgs/applications/audio/plexamp/default.nix:35]: Oh, you put it in alphabetical order, that’s fine too...

[the remark this answers is not in the document]
Comment (Redhawk18) [reply pkgs/applications/audio/plexamp/default.nix:35]: This pr doesn't build anything because I've had to split hairs over multiple pr's. Resolved

**Comment by NobbZ** [comment]
> I know there are whitespace changes, Its automatic by my editor.

Please learn to control this behaviour. There is a lot of manual formatting going on in nixpkgs currently, and if you really can not convince your editor to not introduce unrelated changes, please temporarily switch to `nano`.

**Pull request closed without merging**